File: loops/loop.cs
// // print the table of n number :

// using System;

// class TableOfN
// {
//     static void Main(string[] args)
//     {
//         Console.Write("Enter a number: ");
//         int n = Convert.ToInt32(Console.ReadLine());

//         Console.WriteLine("Table of " + n + ":");
//         for (int i = 1; i <= 10; i++)
//         {
//             Console.WriteLine(n + " x " + i + " = " + n * i);
//         }
//     }
// }


using System;

class TableOfN
{
    static void Main(string[] args)
    {
        Console.Write("Enter a number: ");
        int n = Convert.ToInt32(Console.ReadLine());


        for (int i = 1; i <= n; i++)
        {
            if (i % 2 == 0)
            {
                Console.WriteLine($"\n{i} is even");
            }
            else
            {
                Console.WriteLine($"\n{i} is odd");

            }
        }

    }
}

File: loops/loop.cpp
// print a table of n number:
// #include <iostream>
// using namespace std;
// int main()
// {
//     int n, i;
//     cout << "Enter number: ";
//     cin >> n;
//     for (i = 1; i <= 10; i++)
//     {
//         cout << "\n" << n << " * " << i <<" = "<< n * i;
//     }
//     return 0;
// }

// write even odd number from 0 to n number:

#include <iostream>
using namespace std;
int main()
{
    int n, i;
    cout << "Enter number: ";
    cin >> n;
    for (i = 0; i <= n; i++)
    {
        if (i % 2 == 0)
        {
            cout << "\n"
                 << i << " is even.";
        }
        else
        {
            cout << "\n"
                 << i << " is odd.";
        }
    }
}